app/controllers/spree/payments_controller.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Spree
+  class PaymentsController < Spree::StoreController
```

Comment:
Is there a reason for creating this new controller in the Spree namespace? if it's new and it's not technically Spree code, we can put it under `app/controllers/` instead?

Reply:
Ah right, it maps to the Spree::Payment model. Ignore me :see_no_evil:

Reply:
lol I was just opening it up and wonder why the heck I *did* put it there....

Reply:
Yeah I think we can move it.

Reply:
c71e5037129c43f2431c4ab4037af7cea20f6e88